### DISCUSSION TITLE
Using syncplay with flatpak apps

### DISCUSSION THREAD

**Discussion by arvigeus** [Syncplay/syncplay#528] [Show and tell]
1) Create a `mpv.sh` file with the following content:

```sh
#!/bin/bash
flatpak run io.mpv.Mpv "$@"
```

2) Make the file executable

3) Point syncplay to that file when asks you for a player

Should work with other supported players as well

**Comment by soredake** [comment]
@arvigeus can you add `LD_LIBRARY_PATH=` before flatpak command? It will fix https://github.com/Syncplay/syncplay/issues/536
